src/sentry/receivers/onboarding.py

```diff
@@ -152,6 +175,7 @@ def record_release_received(project, group, event, **kwargs):
                     project_id=project.id,
                     date_completed=timezone.now()
                 )
+                check_for_onboarding_complete(project.organization)
```

Comment:
I'd move all of these calls outside of this `try..except` block and I'd move it to the `else`.